### ISSUE TITLE
Help needed for windows E254

### ISSUE THREAD

**Issue by vakosel** [rafi/vim-config#12]
Hi again!
In the firste place no issues in Linux os.
In windows 10 no errors in cterm but when i open gvim then i get the following 
Error detected while processing Colorscheme Auto commands for "*":
E254 cannot allocate color #333
Even changing colorscheme does not fix the problem .So gvim cannot render the specified color
somewhere . I tried to find it but no luck .
Any help?
thanks in advance

**Comment by rafi** [comment]
Hi. Try to disable `config/theme.vim` by commenting out following line from `vimrc`:
```
call s:source_file('theme.vim')
```

**Comment by vakosel** [comment]
Thanks for your immediate answer .I cannot find anywhere such statement . Neither theme.vim
Can you point me  the exact location?

**Comment by rafi** [comment]
Oh sorry. It's something I haven't committed yet.
Try to disable this [line](https://github.com/rafi/vim-config/blob/8a4b08e274296bd3aebba331395e9fcbfff66c6b/config/general.vim#L239).

**Comment by vakosel** [comment]
it does not seem to work . I commented out the whole last "Theme section " but no luck.
E254  VimEnter AutoCommands for "*" .. Cannot allocate color ..#333
as i mentioned before.Weird ..
Also just to let you know i had to symlink your /plugins/vimrc for installation (make install)  to proceed.Also i had to set afterwards in runtime path  my %HOME%/.vim directory which is actually your git folder due to windows  bu....sh..t. Otherwise when using   :set runtimepath? my %HOME%/.vim directory was not there. All %HOME%/.cache was there  and plugins and everything.It did't recognize whithout symlinking the vimrc the whole directory. I just mention this for portability issues.
In short  this line https://github.com/rafi/vim-config/blob/master/config/vimrc#L18 does not work in windows .
Now that i did that everything is perfect . Apart from gvim  when starting.Afrerwards no problem at all.
Any other idea?

**Comment by rafi** [comment]
Could you try changing `#333` to `#333333` in these [two lines](https://github.com/rafi/vim-config/blob/8a4b08e274296bd3aebba331395e9fcbfff66c6b/config/plugins.vim#L568-L569)?

**Comment by vakosel** [comment]
Case closed my friend ! That did the trick. I checked for 333 in various files and i would do it myself but didn't check in plugins.vim :+1: .
If you automate the above workarounds i did in my previous comment it's good to go in windows too!
Thanks again
